randovania/games/dread/json_data/Artaria.txt

```diff
@@ -5062,12 +5068,12 @@ Extra - asset_id: collision_camera_081
   * Extra - right_shield_def: None
   > Tile Group (SCREWATTACK) 2
       After Artaria - Screw Attack Rotatable
-  > Tile Group (WEIGHT) 3
-      Morph Ball and After Artaria - Screw Attack Rotatable
```

Comment:
Reason behind removing this? I thought you should be able to reach these crumble blocks with morph ball but no screw attack?

Reply:
The door goes to SCREWATTACK 2 and SCREWATTACK 2 goes to WEIGHT 3, therefore this connection is now redundant

Reply:
unless im misunderstanding things